src/containerapp/azext_containerapp/_clients.py

```diff
@@ -25,1491 +35,12 @@
-        sub_id = get_subscription_id(cmd.cli_ctx)
-        request_url = f"{management_hostname}subscriptions/{sub_id}/resourceGroups/{resource_group_name}/providers/Microsoft.App/containerApps/{container_app_name}/authConfigs/{auth_config_name}?api-version={cls.api_version}"
-
-        if "properties" not in auth_config_envelope:  # sdk does this for us
-            temp_env = auth_config_envelope
-            auth_config_envelope = {}
-            auth_config_envelope["properties"] = temp_env
-
-        r = send_raw_request(cmd.cli_ctx, "PUT", request_url, body=json.dumps(auth_config_envelope))
-
-        if no_wait:
-            return r.json()
-        elif r.status_code == 201:
-            request_url = f"{management_hostname}subscriptions/{sub_id}/resourceGroups/{resource_group_name}/providers/Microsoft.App/containerApps/{container_app_name}/authConfigs/{auth_config_name}?api-version={cls.api_version}"
-            return poll(cmd, request_url, "waiting")
-
-        return r.json()
-
-    @classmethod
-    def delete(cls, cmd, resource_group_name, container_app_name, auth_config_name, no_wait=False):
-        management_hostname = cmd.cli_ctx.cloud.endpoints.resource_manager
-        sub_id = get_subscription_id(cmd.cli_ctx)
-        request_url = f"{management_hostname}subscriptions/{sub_id}/resourceGroups/{resource_group_name}/providers/Microsoft.App/containerApps/{container_app_name}/authConfigs/{auth_config_name}?api-version={cls.api_version}"
-
-        r = send_raw_request(cmd.cli_ctx, "DELETE", request_url)
-
-        if no_wait:
-            return  # API doesn't return JSON (it returns no content)
-        elif r.status_code in [200, 201, 202, 204]:
-            request_url = f"{management_hostname}subscriptions/{sub_id}/resourceGroups/{resource_group_name}/providers/Microsoft.App/containerApps/{container_app_name}/authConfigs/{auth_config_name}?api-version={cls.api_version}"
-            if r.status_code == 200:  # 200 successful delete, 204 means storage not found
-                from azure.cli.core.azclierror import ResourceNotFoundError
-                try:
-                    poll(cmd, request_url, "scheduledfordelete")
-                except ResourceNotFoundError:
-                    pass
-                logger.warning('Containerapp AuthConfig successfully deleted')
-        return
-
-    @classmethod
-    def get(cls, cmd, resource_group_name, container_app_name, auth_config_name):
-        management_hostname = cmd.cli_ctx.cloud.endpoints.resource_manager
-        sub_id = get_subscription_id(cmd.cli_ctx)
-        request_url = f"{management_hostname}subscriptions/{sub_id}/resourceGroups/{resource_group_name}/providers/Microsoft.App/containerApps/{container_app_name}/authConfigs/{auth_config_name}?api-version={cls.api_version}"
-
-        r = send_raw_request(cmd.cli_ctx, "GET", request_url)
-        return r.json()
+class WorkloadProfilePreviewClient(WorkloadProfileClient):
```

Comment:
why do we need this client?

Reply:
The WorkloadProfilePreviewClient can is unused, I will remove it.